### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by willbarton**
This change adds our [code of conduct](https://github.com/cfpb/open-source-project-template/blob/master/CODE_OF_CONDUCT.md) to this repository.

This PR has also run `black` to reformat some files, as the existing formatting didn't pass the lint check.

### CODE REVIEW

File: flags/tests/test_urls.py [hunk 1]
```diff
@@ -226,14 +228,18 @@ def test_flagged_url_false_include_fallback_none(self):
...
         with self.assertRaises(Http404):
             self.get_url_response(
                 "/include-fallback-include/other-included-url"
             )
 
     @override_settings(FLAGS={"FLAGGED_URL": [("boolean", False)]})
-    def test_flagged_url_false_include_fallback_include_nonmatching_url(self,):
+    def test_flagged_url_false_include_fallback_include_nonmatching_url(
+        self,
+    ):
```
Comment (Scotchester) [inline flags/tests/test_urls.py:242]: Would Black do this if there wasn't a trailing comma after `self`? (See also the `@override_settings` changes.) If not, I'd prefer to remove the trailing commas and keep them on one line. The primary benefit to me of the trailing commas is to reduce Git diffs when adding/removing items, which is irrelevant when the arguments still fit on one line.

**Review by Scotchester — APPROVED** [review]
One minor question about Black, but it's fine.

File: flags/tests/test_urls.py (hunk 1 above)
Comment (willbarton) [reply flags/tests/test_urls.py:242]: It looks like not. I'll amend that commit to remove any of those changes.

Comment (Scotchester) [reply flags/tests/test_urls.py]: TY!

**Pull request merged**